.gitignore

```diff
@@ -1,2 +1,3 @@
 Example/Pods
+Example/Podfile.lock
```

Comment:
the lock file should be committed to ensure the right versions are always installed.